README.md

```diff
@@ -344,7 +344,28 @@ export class UserPreview {
 
 # Forms
 
-> TODO
+### Angular
+```js
+@Component({
+  selector: 'reactive-form',
+  template: `
+    <div>
+    <form [formGroup]="form" (ngSubmit)="onSubmit(form.value, form.valid)" novalidate>
+      Name: <input type="text" formControlName="name">
```

Comment:
Would be great to add `label` here e.g.
`<label>Name: <input type="text" formControlName="name"></label>`